### ISSUE TITLE
Wrong indentation on anonymous function

### ISSUE THREAD

**Issue by OskarPersson** [ejmr/php-mode#251]
This does not get indented correctly
```
$app->group('/route', function () use ($app) {
    echo 'hello';
});
```

Instead it looks like this
```
$app->group('/route', function () use ($app) {
        echo 'hello';
    });
```

`php-mode-version` says
> PHP mode 1.15.3 of 2015-03-09

**Comment by syohex** [comment]
Which coding style do you use ?

**Comment by OskarPersson** [comment]
PSR-2

**Comment by ejmr** [comment]
Changing `c-offsets-alist` to set both `defun-block-intro` and `defun-close` to `0` produces the desired result, but we'll need to carefully check that those changes do not screw up any other indentation.

**Comment by ejmr** [comment]
Fixed as of commit 52a75c0ec1aa267dc1e362f8eae493225de9e159.
